src/Components/ErrorModal.js

```diff
@@ -1,18 +1,23 @@
-import React, {useContext} from 'react'
-import { Errors } from '../context'
+import React, { useContext } from 'react'
+import { Errors } from '../Contexts/ErrorsProvider'
 import '../Styles/ErrorModal.scss'
 
 const ErrorModal = () => {
   const {errorMessage, setErrorMessage} = useContext(Errors)
   console.log(errorMessage);
+  if(errorMessage){
   return (
     <div className='modal'>
       <article className='modal-box'>
         <p>{errorMessage}</p>
         <button className='modal-btn' onClick={() => setErrorMessage({error: null})}>OK</button>
       </article>
     </div>
-  )
+  )} else {
+    return(
+      <></>
```

Comment:
I like the idea of just returning an empty fragment. This is a really cool way of handling this logic.